STLtoDATTranslator/Header Files/FileWrite.h

```diff
@@ -10,3 +10,5 @@ class Write {
 public:
     void writeFile(const std::string& filename, const std::vector<Triangle>& triangles, const std::vector<double>& uniqueVertices);
```

Comment:
you have used reference operator in parameters, it is a good practice to use them.